site/ja/lite/guide/roadmap.md

```diff
@@ -0,0 +1,75 @@
+# TensorFlow Lite 2019 ロードマップ
+
+**Updated: August 29, 2019**
+
+下記に述べるのは、私たちの2019年の計画のハイレベルの概観です。
+読者はこのロードマップはいつでも、様々な理由で変更することがあり、
+そして下記の順序はいかなる種類の優先度も反映していないことを意識してください。
+原則として、私たちは概ね大部分のユーザーが求めている事柄を優先させており、
+このリストは基本的にそれを反映しています。
+
+私たちはロードマップを4つの分野に区分けしています：
+有用性、性能、最適化、そして可搬性。
+読者はぜひ私たちのロードマップを論評し、
+TF Liteの議論グループもしくはフォーラムでフィードバックしてください。
+
+## 有用性
+
+*   **新しいモデルコンバーター**
+    *   グラフ変換(例、コントロールフロー、条件付き、など)をより上手く行う新しいMLIRベースのTensorFlow Liteコンバーター
+    *   モデル変換の失敗についての診断やデバッグの改善
+*   **拡張されたopsカバレッジ**
+    *   ユーザーフィードバックに基づいた優先度でのopの追加
+*   **TensorFlowのopsをTensorFlow Liteで使う際の改善**
+    *   バイナリ(Android)やCocoapods(iOS)にて利用できるプリビルドのライブラリ
+    *   op strippingによってselect TF opsを使う際のバイナリサイズをより小さく
+*   **LSTM / RNNサポート**
+    *   LSTMとRNNの変換のフルサポート
+*   **前処理と後処理のサポート**
+    *   モデル固有の前処理と後処理のための新しいサポートライブラリ
+    *   共通するプラットフォーム固有機能のためのユーティリティ、例、効率的にassetsからモデルをロードしたりBitmapをテンソルに変換する
+*   **制御フローとオンデバイスの学習**
+    *   制御フロー関連のopsのサポート
+    *   オンデバイスの学習のサポート、特にパーソナライズ転移学習関連
```

Comment:
[Weak Suggestion]

I think `personalization` and `transfer learning` should not be together.